### PR TITLE
Finish AddScheduleCommand

### PR DESCRIPTION
**Pull request by lizekai-richard**

### CODE REVIEW

**Review by myangat0343 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/AddScheduleCommand.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import seedu.address.model.Model;
+import seedu.address.model.module.schedule.Schedule;
+
+/**
+ * Adds a schedule to a module
+ */
+public class AddScheduleCommand extends Command {
+    public static final String COMMAND_WORD = "add schedule";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book. "
```
Comment (leongdl135) [inline src/main/java/seedu/address/logic/commands/AddScheduleCommand.java:20]: Should the MESSAGE_USAGE be changed to `Adds a schedule to ProfNUS`

**Review by leongdl135 — COMMENTED** [review]
LGTM! Currently, does the AddScheduleCommand add the store the schedule in the json file?

**Review by bryanngzh — APPROVED** [review]
LGTM!

**Pull request merged**